### PR TITLE
Configure Beman.Optional26

### PR DESCRIPTION
**Pull request by neatudarius**
* Update root README.md
* Update CODEOWNERS / .gitignore
* Remove unused build scripts & Docker infrastructure ("bld" is unused after #4). ~~Note: This PR will be merged after #4 - required new CI rules to pass if `bld` is removed.~~
* Move some left over paper build related files from root to paper subdirectory.

This partially solves #12 .

### CODE REVIEW

File: papers/P2988/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# P2988
```
Comment (neatudarius) [inline papers/P2988/README.md:1]: Content moved from root README.md

File: .gitignore [hunk 2]
```diff
@@ -22,6 +24,5 @@ compile_commands.json
 /papers/**/.venv.work/
 /papers/**/_minted-optional_ref/
 /papers/**/_minted-optional_ref_wording/
-/.cache/
-*.pyg
-/.build/
+.update-submodules
+.use-docker
```
Comment (steve-downey) [inline .gitignore:28]: Can delete the flag file for docker, too. Harmless to ignore, though.

File: README.md [hunk 3]
```diff
@@ -18,49 +21,54 @@ The README itself is licesed with CC0 1.0 Universal. Copy the contents and incor
...
   - CMake defaults to "Unix Makefiles" on POSIX systems
 
-Compiling the paper requires a working LaTeX installation. See instructions for configuring your system at [C++ Standard Draft Sources](https://github.com/cplusplus/draft/blob/main/README.rst)
+Example of installation on `Ubuntu 24.04`:
+```shell
+# install tools
+apt-get install -y cmake make ninja-build
+
+# example of toolchains
+apt-get install ++-14 gcc-14 clang-18 clang++-18
```
Comment (steve-downey) [inline README.md:42]: g++14 rather than `++-14` ?

**Review by steve-downey — APPROVED** [review]
LGTM
👍

File: README.md (hunk 3 above)
Comment (neatudarius) [reply README.md]: yes, it's a typo :D

File: CITATION.cff [hunk 4]
```diff
@@ -1,7 +1,4 @@
-cff-version: 1.2.0
+cff-version: 1.0.0
 message: "If you use this software, please cite it as below."
-authors:
```
Comment (neatudarius) [inline CITATION.cff:3]: @steve-downey , any idea what / if we should write here?

**Pull request merged**